src/OpenSage.Game/GameWindow.cs

```diff
@@ -77,19 +64,37 @@ public GameWindow(string title, int x, int y, int width, int height, GraphicsBac
                 ResourceBindingModel = ResourceBindingModel.Improved
             };
 
+            var windowCreateInfo = new WindowCreateInfo(x, y, width, height, WindowState.Normal, title);
+
             if (preferredBackend != null)
```

Comment:
I've opened https://github.com/mellinoe/veldrid/pull/65 which will let us remove this `if`.